ingestion/src/metadata/ingestion/source/dashboard/dashboard_service.py

```diff
@@ -274,6 +274,9 @@ def yield_create_request_dashboard_service(self, config: WorkflowSource):
             entity=DashboardService, config=config
         )
 
+    def process_owner(self, dashboard_details: dict):
+        pass
```

Comment:
we could add a debug log here saying that processing the ownership is not supported

Reply:
yes